gemini_test.py

```diff
@@ -138,6 +139,33 @@ def get_email_data(self):
             self.log.warning("Failed to get gemini version as loader instance is not created")
             gemini_version = ""
 
+        try:
```

Comment:
I'm wondering why we are doing this on the gemini_test.py ? and no in a more central place ?
i.e. we have case that we might be running gemini of other cases, wouldn't we ?

Reply:
I'm not sure, I'll check it in this place first, but yeah, might want to put in a more appropriate place, should probably go into normal `ClusterTester` into teardown stage.

Reply:
Moved to `ClusterTester` now, done in teardown stage after the email data call, so should be equivalent.